### DISCUSSION TITLE
Moves that make contact

### DISCUSSION THREAD

**Discussion by supercomputer276** [PokeAPI/pokeapi#905] [Ideas]
One thing I've noticed is that the database doesn't seem to keep track of what moves make contact. While many "invisible" qualities of moves can be intuited from their (Japanese) name, contact doesn't correlate to anything else and feels like something that should be kept track of explicitly. It'd fit right into the meta section.

**Comment by Naramsim** [comment]
Yep we lack that info :( Scraping for bulbapedia and extending the API seems feasible though.

**Reply by Naramsim** [reply]
https://bulbapedia.bulbagarden.net/wiki/Contact

**Comment by Charlotte-br560** [comment]
Keeping track of moves that make contact sounds like a valuable addition to the database. It would indeed provide useful insights, especially for competitive analysis. Adding it to the meta section seems like a logical step.
